sip editor committee/editors_list.md

```diff
@@ -0,0 +1,127 @@
+
+**Rafael Cárdenas**: Rafael is a Staff Software Engineer at Hiro, focused on building and maintaining Hiro’s API infrastructure which includes the Stacks Blockchain API, Ordinals API, Token Metadata API and others, all of which power dozens of apps and wallets built on Stacks and Bitcoin. Rafael has also been an active member and contributor of the Stacks ecosystem since 2021, involved with the development of the core blockchain through the SIP process and participating as a reviewer at the Stacks Accelerator.
+
+**Sleepi**: I am a study coordinator in the medical/pharmacy research with a strong passion for cryptocurrency, having become fascinated with its dynamic nature through day trading. As a community representative, I am committed to educating and supporting others and ensuring fair representation for all members. While valuing privacy, I continue to expand my knowledge in this rapidly-evolving industry and am open to connecting with others to discuss all things crypto. I am an active moderator and support in the official Stacks discord server.
+
+**Tigs / Oli**: tigs / Oli: Yo! The name is tigs and have been a multiplayer pro gamer since the early 90s. Not quite so pro anymore but through this exposure I taught myself Cisco (CCNA) and server administration at a young age. I haven't passed high school (got a GED), have attended college for gaming dev but never completed and have been involved in a few jobs in regards to teaching, inspiring and getting aspiring tech people into the industry as whole. Call me a Jack of all trades. Master of none! I am deeply involved with Horticulture as well (green fingers). I run several bitcoin+lightning nodes, a few miners and am deeply interested in Bitcoin as a saver of humanity, not purely as an investment vehicle. I am also involved with TSOBTC (The School of BTC).
+
+**Werner.btc**: Works for the Hiro Wallet team as support lead, helping users with issues related to the wallet and transactions. Likes to think about ways to improve Stacks, and dapps in the ecosystem.
+
+## About this SIP Editor Committee
+
+The role of the SIP Editors is to identify SIPs in the Draft status that can be transitioned to Accepted status. A SIP editor must be able to vet a SIP to ensure that it is well-formed, that it follows the ratification workflow faithfully, and that it does not overlap with any already-Accepted SIPs or SIPs that have since become Recommended or Ratified.
```

Comment:
Some small grammar changes

```suggestion
The role of a SIP Editor is to identify SIPs in the Draft status and help transition them to Accepted status. A SIP editor must be able to vet a SIP to ensure that it is well-formed, that it follows the ratification workflow faithfully, and that it does not overlap with any already-Accepted SIPs or SIPs that have since become Recommended or Ratified.
```